src/sentry/issues/occurrence_consumer.py

```diff
@@ -264,8 +265,14 @@ def _process_message(
             txn.set_tag("project_id", project.id)
             txn.set_tag("project_slug", project.slug)
 
-            if not features.has("organizations:profile-blocked-main-thread-ingest", organization):
-                metrics.incr("occurrence_ingest.dropped_feature_disabled", sample_rate=1.0)
+            if occurrence_data["type"] not in PROFILE_FILE_IO_ISSUE_TYPES or not features.has(
+                "organizations:profile-blocked-main-thread-ingest", organization
+            ):
```

Comment:
are we using `"organizations:profile-blocked-main-thread-ingest"` as a multifunctional feature flag here? does `ProfileFileIOGroupType` have its own flag?

Reply:
We're using the same feature flag here. It'd be a lot of work at the moment to go through and manually add a new flag for each type. `ProfileFileIOGroupType` is actually the same data from `ProfileBlockedThreadGroupType`, profiling just made it more specific. Not sure if we're even going to get type 2000 anymore.

Reply:
right. got it. just asking for my own understanding mostly